packages/table/src/table.tsx

```diff
@@ -372,7 +372,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const { isRowHeaderShown } = this.props;
         this.validateGrid();
         return (
-            <div className={Classes.TABLE_CONTAINER} ref={this.setRootTableRef} onScroll={this.handleRootScroll}>
+            <div
+                className={classNames(Classes.TABLE_CONTAINER, this.props.className)}
```

Comment:
I prefer to avoid deconstructing unless:

1. ...a variable is used in more than one place in that scope.
1. ...(less importantly) deconstructing helps keep later lines below our linter's `max-line-length`.

That said, consistency is also nice. I'll deconstruct in `columnHeaderCell.tsx`, since that one seems more egregious.